components/Board/index.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react'
+import { StyleSheet, View, TouchableOpacity, Text } from 'react-native'
+import PropTypes from 'prop-types'
+
+export default class Board extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { lists: [{
+            title: "list1",
+            cards: [{text: 'text1', id: '1'},
+                    {text: 'text2', id: '2'}],
+            },
+            {
+                title: "list2",
+                cards: [{text: 'text3', id: '3'},
+                        {text: 'text4', id: '4'}],
+            }
+        ]};
+    }
+
+    render() {
+        return (
+            <View>
+                <Text>{this.props.name}</Text>
+                <View style={styles.boardWrap}>
+                    {this.state.lists.map((l, i) => <List title={l.title} cards={l.cards} key={i}/>)}
```

Comment:
using the array index as the key here is dangerous. prob better to use the list name, and mandate that they're all different